### PR TITLE
Modifications to support maven for osio build

### PR DESCRIPTION
**Pull request by yzainee**
Do not merge now

### CODE REVIEW

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -704,7 +704,7 @@ def post():
                 filepath = filepaths[index]
                 content = manifest_file_raw.read().decode('utf-8')
             # For npm-list.json, we need not verify it as its not going to mercator task
-            if origin != "vscode" and ecosystem not in ("npm", "pypi", "golang"):
+            if origin != "vscode" and resolved_files_exist(filename) is False:
```
Comment (tisnik) [inline bayesian/api_v1.py]: the function `resolved_files_exist(filename)` returns boolean in all cases, so you can use `not resolved_files_exist(filename)`

File: bayesian/api_v1.py [hunk 2]
```diff
@@ -736,16 +736,12 @@ def post():
...
             deps = {}
-            # TODO: don't use "true" and "false" for Boolean values
-            worker_flow_enabled = "false"
-            # TODO This will be changed once we add support for other ecosystems
+            worker_flow_enabled = False
 
-            if resolved_files_exist(manifests) == "true" \
-                    and ecosystem in ("npm", "pypi", "golang"):
-                # This condition for the flow from vscode
+            if resolved_files_exist(manifests) is True:
```
Comment (miteshvp) [inline bayesian/api_v1.py]: this is same as `if resolved_files_exist(manifests):`

File: bayesian/api_v1.py (hunk 1 above)
Comment (miteshvp) [reply bayesian/api_v1.py]: Something like 
`if origin != "vscode" and not resolved_files_exist(filename):`

File: bayesian/api_v1.py [hunk 3]
```diff
@@ -704,7 +704,7 @@ def post():
                 filepath = filepaths[index]
                 content = manifest_file_raw.read().decode('utf-8')
             # For npm-list.json, we need not verify it as its not going to mercator task
```
Comment (miteshvp) [inline bayesian/api_v1.py]: Change this comment as we support more than npm-list.json files

Comment (yzainee) [reply bayesian/api_v1.py]: done

File: bayesian/api_v1.py (hunk 2 above)
Comment (yzainee) [reply bayesian/api_v1.py]: done

File: bayesian/api_v1.py (hunk 1 above)
Comment (yzainee) [reply bayesian/api_v1.py]: done

**Comment by miteshvp** [comment]
LGTM. Please rebase your branch

**Comment by yzainee** [comment]
[test]

**Pull request merged**